pkg/gci/gci.go

```diff
@@ -154,56 +154,63 @@ func LoadFormatGoFile(file io.FileObj, cfg config.Config) (src, dist []byte, err
 	firstWithIndex := true
 
 	var body []byte
-	// order: standard > default > custom
-	if len(result["standard"]) > 0 {
-		for _, d := range result["standard"] {
-			AddIndent(&body, &firstWithIndex)
-			body = append(body, src[d.Start:d.End]...)
-		}
-
-		body = append(body, utils.Linebreak)
-	}
 
-	if len(result["default"]) > 0 {
-		for _, d := range result["default"] {
-			AddIndent(&body, &firstWithIndex)
-			body = append(body, src[d.Start:d.End]...)
+	// if default order sorted sections
+	if !cfg.CustomOrder {
+		orders := map[string]int{
```

Comment:
make it defined in the header of file as `defaultOrder`.

Reply:
done